src/core/core.scale.js

```diff
@@ -744,7 +747,7 @@ module.exports = Element.extend({
 		var isHorizontal = me.isHorizontal();
 
 		var parseFont = helpers.options._parseFont;
-		var ticks = optionTicks.display && optionTicks.autoSkip ? me._autoSkip(me.getTicks()) : me.getTicks();
+		var ticks = optionTicks.autoSkip ? me._autoSkip(me.getTicks()) : me.getTicks();
```

Comment:
I would keep this optimization of not calling `_autoSkip` when ticks are not shown. (especially now, when `autoSkipPadding` is added to those zeroed values, so some invisible skipping could occur depending on config)

Reply:
Done